eng/common/scripts/check-spelling-in-changed-files.ps1

```diff
@@ -51,6 +51,13 @@ foreach ($file in $changedFiles) {
     $changedFilePaths += $file.Path
 }
 
+# The "files" list must always contain a file which exists, is not empty, and is
+# not excluded in ignorePaths. In this case it will be a file with the contents
+# "1" (no spelling errors will be detected)
+$notExcludedFile = (New-TemporaryFile).ToString()
```

Comment:
Interesting approach I was thinking that you were going to try to apply the ignore filters to see if the set it empty but I think this is a reasonable workaround. 

Only thing I don't like is the fact that we are generating a temp file everytime. Instead can we just pick a file we know will always be present. Perhaps this script file or something in the root of the repo?

Reply:
We can't use the script file itself because it's in `eng/common` which is ignored. We could use `README.md` at the root of the repo but a spelling error detection in `README.md` would block a PR that has changes unrelated to that `README.md`.

Reply:
After our chat I think I'm good with creating a temp file as long as we make an attempt to clean it up which you are doing.